### ISSUE TITLE
Several issues concerning the paymium implementation

### ISSUE THREAD

**Issue by RetGal** [ccxt/ccxt#6819]
On paymium `cancel_order` does throw an error:

```
return self.privateDeleteUserOrdersUUIDCancel(self.extend(request, params))
AttributeError: 'paymium' object has no attribute 'privateDeleteUserOrdersUUIDCancel'
```

Another issue is that  `private_get_user_orders({'uuid': order_id})` leads to a permission error (401), while calling `private_get_user_orders() `without param works -  but is very inefficient.

- OS: Linux (Ubuntu)
- Programming Language version: Python 3.6
- CCXT version: 1.26.45

**Comment by kroitor** [comment]
@RetGal 

> On paymium cancel_order does throw an error:

Uploaded a fix for this one, thx for pointing it out!

> Another issue is that private_get_user_orders({'uuid': order_id}) leads to a permission error (401)

Can you plz post your verbose request/response for this one and the code you use to instantiate it (without your keys, of course)?

- https://github.com/ccxt/ccxt/wiki/FAQ#what-is-required-to-get-help

**Comment by kroitor** [comment]
@RetGal 

The first bug is fixed in CCXT version 1.26.51 – it will arrive in 15 minutes.

The second issue is not a bug, please change your method call from:

```Python
private_get_user_orders({'uuid': order_id})
```

to ↓

```Python
private_get_user_orders_uuid({'uuid': order_id})
```

Let us know if you have further difficulties with it. Thx again for your feedback!

**Comment by RetGal** [comment]
@kroitor 
Thanks a lot! I changed my code and will open another issue if it really is one ;)
